ci-operator/step-registry/ipi/conf/azure/ipi-conf-azure-commands.sh

```diff
@@ -23,6 +23,10 @@ elif [[ "${SIZE_VARIANT}" == "large" ]]; then
   master_type=Standard_D16s_v3
 elif [[ "${SIZE_VARIANT}" == "compact" ]]; then
   master_type=Standard_D8s_v3
+elif [[ "${SIZE_VARIANT}" == "vmgenv1" ]]; then
```

Comment:
From human reading habit, I think `vmgenv1` is not about `cluster size`, and the special instance type - `Standard_NP10s` depends on some specific region, personally it will be more safer to create a dedicated ref step.

Reply:
I will update it